tests/resolve_test.rs

```diff
@@ -15,6 +15,7 @@ fn chinese() {
 }
 
 #[test]
+#[ignore]
```

Comment:
These require a local `pnpm i` in `fixtures/pnpm8`. I should've documented this 😅

Reply:
Thank you, I will revert the change